projects/unusable/runners/predict_catboost_v1.py

```diff
@@ -35,14 +35,16 @@ def run(image_path, shape_path, model_path, resolution, n_processes):
         result.append(batch_to_numpy(batch))
     data_frame = pandas.DataFrame.from_records(result)
     probabilities = classifier.predict_proba(Pool(
-            data_frame.drop(['label', 'field_name', 'base_file_name'], axis=1),
-            cat_features=['satellite']
+        data_frame.drop(['label', 'field_name', 'base_file_name'], axis=1),
+        cat_features=['satellite']
     ))[:, 1]
     result = pandas.DataFrame(.0, index=labels.index, columns=labels.columns)
     for probability, field_name, base_file_name in zip(
         probabilities, data_frame['field_name'], data_frame['base_file_name']
     ):
         result.loc[base_file_name, field_name] = probability
+
+    result = result.rename({s: s + '_rgb' for s in labels.index}, axis=0)
```

Comment:
Do we need this `_rgb`? `evaluate.py` script probably depends on the result of this script in old format.